### ISSUE TITLE
language can't be changed

### ISSUE THREAD

**Issue by ziin1995** [volkanceylan/Serenity#827]
Hi, Volkanceylan

I really appreciate your work, Serenity is the best framework I have seen ever.
I am going make a back-office system for my company and everything is fine except multi-language.

I am preparing two language set, Korean and English.
In my development environment, default language is Korean.
When I change to English at the general settings, it is not work. 
Language setting is changed to Korean automatically.

Have you any experience like my case? 

There are 3 files in my App_data/texts directory.
user.texts.en.json, user.texts.invariant.json and user.texts.ko.json

I hope you could give me some hint.

**Comment by volkanceylan** [comment]
Check cookies and language key you set for Korean in languages page

**Comment by volkanceylan** [comment]
It should be two letter ko lowercase

**Comment by ziin1995** [comment]
Thanks your answer.

I sought cookies and found the language preference is 'en'.
Meanwhile, general settings is set to 'Korean'  which I made as 'ko' / 'Korean'.

I think it is not cookies problem. I've checked 3 types of browser  chrome, firefox and MS edge.
In all of those browsers I can not change the language setting from Korean to English.

I'll reset my system and dig more.

If I found any clue of it, I'll report here.
